Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/clientmetadata/ClientMetadataDonor.kt

```diff
@@ -0,0 +1,52 @@
+package de.rki.coronawarnapp.datadonation.analytics.modules.clientmetadata
+
+import de.rki.coronawarnapp.appconfig.AppConfigProvider
+import de.rki.coronawarnapp.datadonation.analytics.modules.DonorModule
+import de.rki.coronawarnapp.nearby.ENFClient
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.PpaData
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ClientMetadataDonor @Inject constructor(
+    private val clientVersionParser: ClientVersionParser,
+    private val clientVersionWrapper: ClientVersionWrapper,
+    private val appConfigProvider: AppConfigProvider,
+    private val enfClient: ENFClient
+) : DonorModule {
+
+    override suspend fun beginDonation(request: DonorModule.Request): DonorModule.Contribution {
+        val config = appConfigProvider.getAppConfig()
```

Comment:
We should use `configProvider.latestConfig` here. I don't think a 5 hour old config would be a problem here and `Analytics` will already attempt a refresh when checking for conditions.

Reply:
Sounds good 👍